src/main/java/com/egen/AppInitialize.java

```diff
@@ -20,6 +22,7 @@
 
 	@Override
 	protected String[] getServletMappings() {
-		return new  String[]{};
+		log.info("asd");
+		return new  String[]{"/"};
```

Comment:
it is a good pratice to prefix the context with /api

Reply:
Yes, thank you for the feedback. Will keep that in mind for future projects.